tests/update/patch_before_migration.pm

```diff
@@ -58,6 +58,7 @@ sub patching_sle {
         # Update origin system on zVM that is controlled by autoyast profile and reboot is done by end of autoyast installation
         # So we skip reboot here after fully patched on zVM to reduce times of reconnection to s390x
         if (!get_var('UPGRADE_ON_ZVM')) {
+            assert_script_run 'sync', 600;
```

Comment:
I don't understand why `sync` would change anything. Isn't the reboot doing a sync implicitly? What does this call change then?

But I am fine to have this merged anyway, I just don't understand

Reply:
Hi Oliver,
    I add sync before reboot just to decrease the time that 'sync' using in reboot.

Reply:
Perform `sync` ahead of `reboot` to flush filesystem buffers that probably reduce time of sync during rebooting, in the case we just don't want to always enlarge timeout of wait_boot. I also think it may not resolve the issue completely, since it is not harmful to other tests, I would merge it to see consequence.

Reply:
agreed, very cool idea. I would have preferred a comment in code describing your good explanation. Maybe you would consider still adding this in a follow-up PR

Reply:
comment pr: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4732